### PR TITLE
Fix/error on submit

### PR DESCRIPTION
**Pull request by emnlpstr**
fixed error on form submit

## Checklist

- [x] I followed the indications in the [CONTRIBUTING](CONTRIBUTING.md)
- [x] The documentation related to the proposed change has been updated accordingly (also comments in code).
- [ ] Have you written new tests for your core changes, as applicable?
- [ ] Have you successfully ran tests with your changes locally?
- [x] Ready for review! :rocket:

### CODE REVIEW

**Review by lollomag — APPROVED** [review]

File: contents/errorsLabels.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+errors:
+  errorMatch: "Il valore inserito non è valido"
```
Comment (sebbalex) [inline contents/errorsLabels.yml:2]: I'm not particularly convinced about this file, I'd move this content in specific page file

File: src/components/modal/ModalMessage.jsx [hunk 2]
```diff
@@ -571,14 +571,14 @@ export const ModalMessage = () => {
...
-            <img className={classes.spinner} src="/assets/spinner.gif"></img>
+            <img className={classes.spinner} src="/assets/spinner.gif" alt="spinner"></img>
           </div>
         </ModalFooter>
       </Modal>
       <div className="container test-docs">
         <div className="row">
           <div className="col-12 col-md-6">
-            <div className={classes.notification} role="alert" aria-labelledby="not2dms-title" id="not2dms2">
+            {/* <div className={classes.notification} role="alert" aria-labelledby="not2dms-title" id="not2dms2">
```
Comment (sebbalex) [inline src/components/modal/ModalMessage.jsx:581]: remove please

File: src/layouts/Header.jsx [hunk 3]
```diff
@@ -379,7 +454,7 @@ const NavHeader = () => {
                   <span className="font-weight-semibold">{internalLinks.support.label}</span>
                 </Link>
               </NavItem>
-              <NavItem className={classes.updatesBtn} active>
+              {/* <NavItem className={classes.updatesBtn} active>
```
Comment (sebbalex) [inline src/layouts/Header.jsx:457]: same here

File: src/pages/updates.jsx [hunk 4]
```diff
@@ -0,0 +1,5 @@
+import React from 'react';
```
Comment (sebbalex) [inline src/pages/updates.jsx:1]: are we sure this is the correct name for this page?

File: src/layouts/AssistenzaPage.jsx [hunk 5]
```diff
@@ -0,0 +1,722 @@
+/* eslint-disable react/no-unescaped-entities */
+/* eslint-disable sonarjs/no-duplicate-string */
+/* eslint-disable react-hooks/exhaustive-deps */
+/* eslint-disable sonarjs/no-all-duplicated-branches */
+/* eslint-disable sonarjs/no-identical-functions */
+/* eslint-disable sonarjs/cognitive-complexity */
+/* eslint-disable max-lines-per-function */
+/* eslint-disable arrow-body-style */
```
Comment (sebbalex) [inline src/layouts/AssistenzaPage.jsx:8]: These ignore lines relax too much our linting checks, please consider to remove them.

File: src/layouts/AssistenzaPage.jsx [hunk 6]
```diff
@@ -0,0 +1,722 @@
...
+                      id="argument-select"
+                      inputId="argument-select-input"
+                      dataRefer="00N3N00000GCzFR"
+                      options={selectArgument}
+                      onChange={handleArgument}
+                      placeholder={selectPlaceholder}
+                      aria-label={selectPlaceholder}
+                      aria-describedby="mandatory-label"
+                      /* aria-invalid={errors.representative && 'true'}
+                    aria-labelledby={errors.representative && 'error-represent'} */
```
Comment (sebbalex) [inline src/layouts/AssistenzaPage.jsx:559]:
```suggestion

```

File: src/layouts/AssistenzaPage.jsx [hunk 7]
```diff
@@ -0,0 +1,722 @@
...
+                      id="measure-select"
+                      inputId="measure-select-input"
+                      dataRefer="00N3N00000GCzFW"
+                      options={selectMeasure}
+                      onChange={handleMeasure}
+                      placeholder={selectPlaceholder}
+                      aria-label={selectPlaceholder}
+                      aria-describedby="mandatory-label"
+                      /* aria-invalid={errors.representative && 'true'}
+                    aria-labelledby={errors.representative && 'error-represent'} */
```
Comment (sebbalex) [inline src/layouts/AssistenzaPage.jsx:595]:
```suggestion

```

File: src/layouts/AssistenzaPage.jsx [hunk 8]
```diff
@@ -0,0 +1,722 @@
...
+              id="assistance-form"
+              target="formFrame"
+              onSubmit={formHandler}
+            >
+              <input
+                type="hidden"
+                name="captcha_settings"
+                value='{"keyname":"DEV1","fallback":"true","orgId":"00D3N0000004K3l","ts":""}'
+              />
+              <input type="hidden" name="debugEmail" value="mattia.g.puggioni@accenture.com" />
```
Comment (sebbalex) [inline src/layouts/AssistenzaPage.jsx:511]: we should pay attention here

File: src/layouts/UpdatesPage.jsx [hunk 9]
```diff
@@ -0,0 +1,575 @@
+/* eslint-disable react/no-unescaped-entities */
+/* eslint-disable max-lines-per-function */
+/* eslint-disable sonarjs/cognitive-complexity */
+import React, { useState, useEffect } from 'react';
+// import PropTypes from 'prop-types';
```
Comment (sebbalex) [inline src/layouts/UpdatesPage.jsx:5]:
```suggestion
```

**Review by sebbalex — CHANGES_REQUESTED** [review]

**Review by sebbalex — APPROVED** [review]

File: src/pages/updates.jsx (hunk 4 above)
Comment (emnlpstr) [reply src/pages/updates.jsx]: I already opened an other PR where I fixed this

**Review by sebbalex — APPROVED** [review]

**Pull request merged**